lib/_getFunctionalInterface.ts

```diff
@@ -7,6 +7,7 @@ export function getFunctionalInterface(
 ): InsightsClient {
   return (functionName, ...functionArguments) => {
     if (functionName && isFunction((instance as any)[functionName])) {
+      // @ts-ignore
```

Comment:
If this is meant to stay, we could use `@ts-expect-error` and explain why it's needed